src/api-v1.js

```diff
@@ -70,100 +70,106 @@ api.declare({
     // Intentionally vague because we don't want to reveafollowRedirects
     // our configuration too much
     let msg;
+    let code = 503; // default is that something is temporarily wrong
```

Comment:
Good call.  In commit 7e8ec9b1f7fa6e11a13bb38aeae284a5da66ccd9, I changed this to only do the validation when the file *might* be copied, instead of on every request.